src/ui/components/Library/Team/View/TestRuns/Overview/RunResults.tsx

```diff
@@ -17,11 +17,54 @@ import {
   useFileNameTree,
 } from "ui/components/Library/Team/View/TestRuns/Overview/useFileNameTree";
 import { TestRunsContext } from "ui/components/Library/Team/View/TestRuns/TestRunsContextRoot";
-import { TestGroup } from "ui/utils/testRuns";
+import { TestGroup, TestGroups } from "ui/utils/testRuns";
 
 import { TestResultListItem } from "./TestResultListItem";
 import styles from "../../../../Library.module.css";
 
+function hasDuplicateRecordings(fileNameToTests: TestGroup["fileNameToTests"]) {
```

Comment:
When we add temporary code, it's nice to have a Linear issue in place (as well as a "TODO <Linear-issue>" comment) to make sure we don't forget to clean it out.